UnitTestProject1/UnitTestProject1/Task_20.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Chrome;
+
+
+namespace UnitTestProject1
+{
+    [TestClass]
+    public class Task_20
+    {
+        
+        private static String URL = "https://192.168.100.26/";
+        private static IWebDriver driver;
+        private string ExpectedURL = "https://192.168.100.26/Home/Index";
+        public void Login(string username, string password)
+        {
+           driver.Navigate().GoToUrl(URL);
+           driver.FindElement(By.CssSelector("input[id=Username]")).SendKeys(username);
+           driver.FindElement(By.Id("Password")).SendKeys(password);
+          // driver.FindElement(By.Id("SubmitButton")).Click();
+            driver.FindElement(By.TagName("button")).Click();
+        }
+
+        [TestInitialize]
+        public void initialize()
+        {
+           driver = new ChromeDriver();
+    }
```

Comment:
Code formatting, find hotkey for auto-formatting in your IDE. Start and end '{' '}' must be on the same vertical